### PR TITLE
added registrant_columns

### LINKED ISSUE #8225: [UI] - Display registrants for models, components and relationships
#### Current Behavior

We do not show the registrants of meshmodels. 

#### Desired Behavior

registrant

#### Mockups

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Freedisch**
**Notes for Reviewers**

This PR fixes #8225 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Freedisch** [comment]
Not yet ready, missing a few updates

**Review by leecalcote — CHANGES_REQUESTED** [review]
@sudhanshutech @Freedisch there seems to be a missing colspan; the row border under duplicates doesn't extend to registrants.

**Comment by leecalcote** [comment]
@Freedisch will you provide an example response of from the API that shows all duplicates?

**Comment by Freedisch** [comment]
> @Freedisch will you provide an example response of from the API that shows all duplicates?

I don't think duplicates are being returned in the response, but I will update my Meshery version and check

File: ui/components/MeshModelComponent.js [hunk 1]
```diff
@@ -326,6 +326,28 @@ const MeshModelComponent = ({ view, classes }) => {
       },
 
     },
+    {
+      name : "registrant",
+      label : "Registrant",
+      options : {
+        sort : view === COMPONENTS || view === RELATIONSHIPS ? true : false,
```
Comment (leecalcote) [inline ui/components/MeshModelComponent.js:333]: @aabidsofi19 thoughts?

Comment (leecalcote) [reply ui/components/MeshModelComponent.js:333]: @sudhanshutech where is data table on your revision list?

Comment (aabidsofi19) [inline ui/components/MeshModelComponent.js:333]:
```suggestion
        sort : view === COMPONENTS || view === RELATIONSHIPS ,
```
the ternary check is not required

File: ui/components/MeshModelComponent.js [hunk 2]
```diff
@@ -326,6 +326,28 @@ const MeshModelComponent = ({ view, classes }) => {
...
+        searchable : view === RELATIONSHIPS ? false : true,
```
Comment (aabidsofi19) [inline ui/components/MeshModelComponent.js:334]:
```suggestion
        searchable :  view !== RELATIONSHIPS 
```

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

File: ui/components/MeshModelComponent.js [hunk 3]
```diff
@@ -326,6 +326,28 @@ const MeshModelComponent = ({ view, classes }) => {
       },
 
     },
+    {
+      name : "registrant",
+      label : "Registrant",
+      options : {
+        sort : view === COMPONENTS || view === RELATIONSHIPS ,
+        searchable :  view !== RELATIONSHIPS
```
Comment (sudhanshutech) [inline ui/components/MeshModelComponent.js:334]: @leecalcote i guess a `,` missed here

**Review by sudhanshutech — APPROVED** [review]

**Comment by Chadha93** [comment]
@sudhanshutech Will yo please takeover this PR?

**Comment by sudhanshutech** [comment]
> @sudhanshutech Will yo please takeover this PR?

okay

**Pull request closed without merging**